get-resource.sh

```diff
@@ -63,6 +63,11 @@ else
       unxz "$RHCOS_IMAGE_FILENAME_RAW"
     fi
 
+    if [ -n "$IP_OPTIONS" ] ; then
+        BOOT_DISK=$(virt-filesystems -a "$RHCOS_IMAGE_FILENAME_QCOW" -l | grep boot | cut -f1 -d" ")
```

Comment:
probably should add the `LIBGUESTFS_BACKEND=direct` to that line as well
otherwise, it defaults to using libvirtd